### ISSUE TITLE
DCM2niix failure under slicer

### ISSUE THREAD

**Issue by muratmaga** [SlicerDMRI/SlicerDcm2nii#6]
I have been trying to fix SlicerDCM2NIix extension to fix the dataset referenced in this thread: 
https://discourse.slicer.org/t/volume-rendering-produces-block-of-white-noise/10572

I encountered a strange issue. The dcm2niix extension fails under through the SLicer at the final step of conversion (Slicerdcm2nii_error.txt), but works fine if invoked from the command line using the same conversion parameters (dcm2niix_standalone.txt). This on windows 10 (my home computer, I don't have the exact version of slicer, but a preview within the last 1-2 weeks). 

[dcm2niix_standalone.txt](https://github.com/SlicerDMRI/SlicerDcm2nii/files/4345169/dcm2niix_standalone.txt)
[Slicerdcm2nii_error.txt](https://github.com/SlicerDMRI/SlicerDcm2nii/files/4345170/Slicerdcm2nii_error.txt)

@lassoan @pieper @fedorov

**Comment by fedorov** [comment]
@afshinmessiah, is this the same issue that you encountered?

**Comment by afshinmessiah** [comment]
Yeah, pretty much the same. On slicer, it works for some cases but for the most cases, it does nothing (no error/warning).

**Comment by pieper** [comment]
@afshinmessiah  - do you have a test case that fails consistently?  The one @muratmaga pointed to is huge, so it would be good to have a smaller case for testing.

**Comment by afshinmessiah** [comment]
@piper I looked into it and dug it out a little bit. I guess, I figured the root cause that behavior. The script switches on the `-i` flag: `dcm2niix.exe  -i, y, ..` which ignores the derived images. Those images which were not opened were derived images and filtered out on slicer. Removing the flag I could open all files. Checked the error for @muratmaga   and I think that is a different error (sort of like mixed series due to `Slice re-ordering resolved inter-slice distance variability`).

**Comment by pieper** [comment]
Thanks for the update @afshinmessiah.
